.github/workflows/test-db.yml

```diff
@@ -0,0 +1,25 @@
+name: '!Test Databases'
```

Comment:
can remove this one right? it's more of a local dev thing

Reply:
yeah good catch thanks